indico/modules/events/editing/client/js/components/EditingView/EditingView.module.scss

```diff
@@ -0,0 +1,28 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@import 'base/palette';
+
+.editing-view {
+    display: flex;
+
+    .contents {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
```

Comment:
I think with flex you can just use `flex: 1`?

Reply:
For what purpose?

Reply:
To make it grow/shrink appropriately thus this is more `flex`y than using `width`